src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1485,8 +1506,17 @@ impl DdlController {
                             dropping_sink_id: None,
                             updated_sink_catalogs,
                         },
+                        drop_table_connector_ctx.as_ref(),
                     )
                     .await?;
+                if let Some(drop_table_connector_ctx) = &drop_table_connector_ctx {
+                    // drop table associated source, we have already drop the objects in catalog above and need to run command to clean up the hummock entry.
```

Comment:
What is "hummock entry"?

Reply:
This should be a comment left over from the incorrect implementation of trigger barrier before, need to update it.